sync/src/main/java/tech/pegasys/teku/sync/forward/multipeer/MultipeerCommonAncestorFinder.java

```diff
@@ -32,27 +31,32 @@
   private final RecentChainData recentChainData;
   private final CommonAncestor commonAncestorFinder;
   private final EventThread eventThread;
+  private final Spec spec;
 
   @VisibleForTesting
   MultipeerCommonAncestorFinder(
       final RecentChainData recentChainData,
       final CommonAncestor commonAncestorFinder,
-      final EventThread eventThread) {
+      final EventThread eventThread,
+      final Spec spec) {
     this.recentChainData = recentChainData;
     this.commonAncestorFinder = commonAncestorFinder;
     this.eventThread = eventThread;
+    this.spec = spec;
   }
 
   public static MultipeerCommonAncestorFinder create(
-      final RecentChainData recentChainData, final EventThread eventThread) {
+      final RecentChainData recentChainData, final EventThread eventThread, final Spec spec) {
     return new MultipeerCommonAncestorFinder(
-        recentChainData, new CommonAncestor(recentChainData), eventThread);
+        recentChainData, new CommonAncestor(recentChainData), eventThread, spec);
   }
 
   public SafeFuture<UInt64> findCommonAncestor(final TargetChain targetChain) {
     eventThread.checkOnEventThread();
     final UInt64 latestFinalizedSlot =
-        compute_start_slot_at_epoch(recentChainData.getFinalizedEpoch());
+        recentChainData
+            .getFinalizedEpoch()
+            .times(spec.getSlotsPerEpoch(recentChainData.getFinalizedEpoch()));
```

Comment:
There should be a more direct transaction of this, I think something like:

```suggestion
        spec.computeStartSlotAtEpoch(recentChainData.getFinalizedEpoch());
```